repos/exawind/packages/tioga/package.py

```diff
@@ -20,8 +20,8 @@ def cmake_args(self):
         spec = self.spec
         options = super(Tioga, self).cmake_args()
 
-        if "dev_path" in spec:
-            options.append(self.define("CMAKE_EXPORT_COMPILE_COMMANDS",True))
+        if spec.satisfies("dev_path=*"):
+            cmake_options.append(self.define("CMAKE_EXPORT_COMPILE_COMMANDS",True))
```

Comment:
Is this right? Should is be options?

Reply:
You're right. I just copied it from the nalu package and didn't check it.
```suggestion
            options.append(self.define("CMAKE_EXPORT_COMPILE_COMMANDS",True))
```